src/main/java/vlad/petrovskyi/internetshop/controllers/shoppingcart/ShoppingCartController.java

```diff
@@ -0,0 +1,31 @@
+package vlad.petrovskyi.internetshop.controllers.shoppingcart;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import vlad.petrovskyi.internetshop.lib.Injector;
+import vlad.petrovskyi.internetshop.service.ShoppingCartService;
+
+public class ShoppingCartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+
+    private static final Injector injector = Injector.getInstance("vlad.petrovskyi.internetshop");
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.setAttribute("products",
+                shoppingCartService.getAllProducts(shoppingCartService.getByUserId(USER_ID)));
+        request.setAttribute("sum", shoppingCartService.getAllProducts(
+                shoppingCartService.getByUserId(USER_ID))
+                .stream()
+                .map(p -> p.getPrice().longValue())
+                .reduce(Long::sum).orElse(0L));
```

Comment:
how about just pass List<Product> as attribute? and print it in you jsp using jstl

Reply:
I already pass it as attribute - this attribute is a sum, which calculates how much money a user already has to pay for the products in his shopping cart